package.json

```diff
@@ -4,27 +4,22 @@
   "description": "A Social platform for the creative at heart",
   "main": "index.js",
   "scripts": {
+    "start": "node src/bin/www",
+    "devstart": "babel-watch src/index",
+    "build": "rm -rf dist && mkdir dist && babel src -s -d dist",
     "test": "echo \"Error: no test specified\" && exit 1"
   },
   "author": "Andela Simulations Programme",
   "license": "MIT",
   "dependencies": {
-    "body-parser": "^1.18.3",
-    "cors": "^2.8.4",
-    "dotenv": "^6.0.0",
-    "ejs": "^2.6.1",
-    "errorhandler": "^1.5.0",
-    "express": "^4.16.3",
-    "express-jwt": "^5.3.1",
-    "express-session": "^1.15.6",
-    "jsonwebtoken": "^8.3.0",
-    "method-override": "^2.3.10",
-    "methods": "^1.1.2",
-    "mongoose": "^5.2.2",
-    "mongoose-unique-validator": "^2.0.1",
-    "passport": "^0.4.0",
```

Comment:
Does this task require you to remove all these dependencies, passport, in particular, is currently used in the app for authentication, is there a reason why we should remove it while configuring babel?

Reply:
We won't be using them on the project, they will be replaced. I also prefer a fresh installation, since the project is still new.

Reply:
I'd rather you focus on the task at hand which is to configure babel.

Reply:
Resolved